src/main/java/picard/fingerprint/CrosscheckFingerprints.java

```diff
@@ -549,6 +579,103 @@ private void remapFingerprints(final Map<FingerprintIdDetails, Fingerprint> fpMa
         });
     }
 
+    /**
+     * Inspects the contents of sampleMapFile building a map of Sample->Sample.
+     * Checks for sanity, and then replaces in the fpMap,
+     */
+    private void remapFingerprintsFromFiles(final Map<FingerprintIdDetails, Fingerprint> fpMap, final File sampleMapFile) {
+        final Map<String, String> sampleMap = getStringStringMap(sampleMapFile, "INPUT_SAMPLE_FILE_MAP").entrySet().stream()
+                .collect(Collectors.toMap(e -> {
+                    try {
+                        return IOUtil.getPath(e.getValue()).toUri().toString();
```

Comment:
The use of `toUri()` here is a bit unfortunate as `getPath()` already constructs a URI but doesn't return it. Maybe `IOUtil.getUri()` would be helpful to have.

Reply:
perhaps you could suggest a htsjdk PR? :-)